### PR TITLE
Support Commas in Widget Numbers

### LINKED ISSUE #4: Feat: format all numbers with commas.
**Is your feature request related to a problem? Please describe.**
With covid-19 cases at an all time high and a total of 34M worldwide cases ( at time of writing ), It gets difficult to read the such large numbers in widgets.

**Describe the solution you'd like**
The numbers should be well formatted with commas at every thousandth place. This makes interpretation easier and improves user experience.  
eg: 987654321 -> 987,654,321

**Describe alternatives you've considered**
N/A
**Additional context**
N/A

### PR DESCRIPTION
**Pull request by Ravi61**
This PR aims to resolve https://github.com/aaryankotharii/Corona-Widget/issues/4

### CODE REVIEW

**Comment by aaryankotharii** [comment]
Hey @Ravi61 , great work with solving this issue. Since you have created an extension to solve this, how about we move the extension to the `Extensions` folder. you can add a new file named `Number.swift` and add your extension there.

File: Corona Widget/Widget/View/LargeWidgetBlock.swift [hunk 1]
```diff
@@ -52,7 +52,15 @@ struct countStack : View {
                         .aspectRatio(contentMode: .fit)
                 } //: IF
             } //: GROUP
-            Text((isActive ? "New " : "Total ") + "\(name) : \(Int(total))").bold()
+            Text((isActive ? "New " : "Total ") + "\(name) : \(total.withCommas())").bold()
         } //: HSTACK
     } //: BODY
 }
+
+extension Double {
```
Comment (aaryankotharii) [inline Corona Widget/Widget/View/LargeWidgetBlock.swift]: we can shift this , as stated above.

**Review by aaryankotharii — CHANGES_REQUESTED** [review]

File: Corona Widget/Widget/View/LargeWidgetBlock.swift (hunk 1 above)
Comment (Ravi61) [reply Corona Widget/Widget/View/LargeWidgetBlock.swift]: done

**Comment by aaryankotharii** [comment]
perfect @Ravi61! Thank you for your contribution and feel free to create issues/prs on this repository.
Approved.

**Review by aaryankotharii — APPROVED** [review]

**Pull request merged**